tasks/chimp.js

```diff
@@ -20,8 +20,12 @@ module.exports = function(grunt) {
     // Make this task async
     var done = this.async();
     var options = this.options();
+    var chimpExecutable = '/node_modules/.bin/chimp';
+    if (process.platform === 'win32') {
```

Comment:
With node win32 represents both windows 32 bit and windows 64 bit.

An alternate if you'd prefer would be to do a regex match:
` /^win/.test(process.platform);`
^which would (hopefully) support all future versions of node if they include more OS versions of windows, but I don't know of any plans to change from win32